uint16 motor_left
uint16 motor_right
uint8 rake_flags
uint8 status_motors
uint8 status_charger
uint8 status_additions
